lottery/src/main/java/com/example/lottery/app/Loto6Service.java

```diff
@@ -0,0 +1,10 @@
+package com.example.lottery.app;
+
+import java.util.List;
+
+public class Loto6Service {
+
+    public List<String> pick() {
```

Comment:
webflux 를 사용한다면 이렇게 가는게 맞음!
```suggestion
    public Flux<String> pick() {
```

Reply:
Flux : 0 ~ N 개의 데이터
Mono : 0 ~ 1 개의 데이터
군

Reply:
오늘 열심히 하넼ㅋㅋㅋㅋㅋLGTM!ㅋㅋㅋ